<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  rgba(245, 221, 178, 1.0) ;</string>
  </property>
  <widget class="QLineEdit" name="leEmail">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;
border-bottom: 3px solid rgba(155, 88, 43, 1.0);
border-right: 3px solid  rgba(155, 88, 43, 1.0);</string>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNumero">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;
border-bottom: 3px solid rgba(155, 88, 43, 1.0);
border-right: 3px solid  rgba(155, 88, 43, 1.0);</string>
   </property>
   <property name="placeholderText">
    <string>Numero de empleado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIniciar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(155, 88, 43, 1.0);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Iniciar sesion</string>
   </property>
  </widget>
  <widget class="QLabel" name="mensaje">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>490</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(155, 88, 43, 1.0);;
</string>
   </property>
   <property name="text">
    <string>Mensaje</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../BookingRoom.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelLink">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>580</y>
     <width>59</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;
&lt;head/&gt;
&lt;body&gt;
&lt;p&gt;&lt;a href=&quot;registro&quot;&gt;registrate&lt;/a&gt;&lt;/p&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
